testing.rst

```diff
@@ -284,8 +284,9 @@ document::
 
         // Assert that the response is a redirect to /demo/contact
         $this->assertTrue(
-            $client->getResponse()->isRedirect('/demo/contact'),
-            'response is a redirect to /demo/contact'
+            $client->getResponse()->isRedirect('/demo/contact')
```

Comment:
I'm afraid that's too much for this PR. Even if I agree with your idea (although I use a different implementation) I'd say that change would need a discussion first. Thanks!